ui/src/components/InfoBar.tsx

```diff
@@ -37,6 +37,11 @@ export default function InfoBar() {
   }, [rpcDataChannel]);
 
   const isCapsLockActive = useHidStore(state => state.isCapsLockActive);
+  const isShiftActive = useHidStore(state => state.isShiftActive);
```

Comment:
State us kept in sync between the physical keyboard (in WebRTCVideo.tsx) and the virtual keyboard (in VirtualKeyboard.tsx)